src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -116,6 +105,7 @@ private static string GetItemKind(DeclaredSymbolInfo declaredSymbolInfo)
                 case DeclaredSymbolInfoKind.Interface:
                     return NavigateToItemKind.Interface;
                 case DeclaredSymbolInfoKind.Constructor:
+                case DeclaredSymbolInfoKind.ExtensionMethod:
```

Comment:
yes.  a C# "public static void Foo(this object o)" is 'declared' as an extension method.

Note: with this change VB extension methods will no longer show up as an extension method in NavTo.  I can do some work to attempt add a pseudo-heuristic to look for ```<....Extension>``` attributes on VB methods.  That's about the best we can do since we're now syntactic here.